src/sentry/replays/endpoints/organization_replay_index.py

```diff
@@ -32,7 +42,21 @@ def get_replay_filter_params(self, request, organization):
 
         return filter_params
 
+    @extend_schema(
+        operation_id="Query for a List of Replays",
```

Comment:
Endpoints that return a list are usually titled `List ...` (eg: [List an Organization's Teams](https://docs.sentry.io/api/teams/list-an-organizations-teams/)). Ideally we would do the same here, but I'm unfamiliar with how replays work. Is the returned list of replays tied to some project or something else that we can include in the title?

Reply:
I'll change it to something like "List an Organization's Replays" since replays are associated by org slug